src/docs/css-helpers/box-alignment.md

```diff
@@ -0,0 +1,170 @@
+# Box Alignment
+
+Responsive box-alignment helper classes allow aligning the content across both
+axes by setting the `align-items` and `justify-content` CSS properties.
+
+Use the box-alignment classes **on flex or grid containers,** they will not work
+elsewhere.
+
+👉 If you need to quickly create a flex container, there is the
+[`.d-flex`](/docs/css-helpers/display) helper class at your disposal.
+
+📖 Read more about [flexbox] and [grid] layout concepts.
+
+## Naming System
+
+The classes are named using the format `[alignment]-[value]` for `xs` and
+`[alignment]-[breakpoint]-[value]` for `sm`, `md`, `lg`, `xl`, `x2l`, and
+`x3l` [breakpoints](/docs/foundation/breakpoints).
+
+Where `alignment` is one of:
+
+- `align-items`
+- `align-self`
+- `justify-content`
+- `justify-self`
+
+For `align-items` and `align-self` the value can be:
+
+- `center`
+- `start`
+- `flex-start`
+- `end`
+- `flex-end`
+- `baseline`
+- `stretch`
+
+For `justify-content` the value can be:
+
+- `center`
+- `start`
+- `flex-start`
+- `end`
+- `flex-end`
+- `space-between`
+
+For `justify-self` the value can be:
+
+- `center`
+- `start`
+- `end`
+- `baseline`
+- `stretch`
+
+👉 The difference between `start`/`flex-start` and `end`/`flex-end` is that the
+prefixed variants are intended for flexbox while the versions without prefix
+only work inside grid layout.
+
+📖 [Read more about CSS Box Alignment concepts.][box-alignment]
+
+## Horizontal (Main-Axis) Alignment (Justification)
```

Comment:
Error in preview:
![image](https://user-images.githubusercontent.com/10213870/209104867-797555e6-a733-4a60-90d0-2d00c3f54b46.png)

Reply:
fixed